app/forms/provider_interface/offer_wizard.rb

```diff
@@ -170,10 +170,12 @@ def structured_conditions
       ske_conditions
     end
 
+    delegate :name, to: :subject, prefix: true
+
   private
 
     def subject
-      course_option.course.subjects.first
+      course_option&.course&.subjects&.first || course.subjects.first
```

Comment:
presumably needed to deal with the case where `course_option` might be nil. but do we really want to suppress the error?

Reply:
I think it's not really an error because `course_option` should be `nil` if the provider is between changing the course and selecting the course option.

Reply:
ah, understood, thanks